tools/tinker/tinker2lmp.py

```diff
@@ -227,7 +227,11 @@ def output(self,outfile):
       print(i+1,label[i],x[i],y[i],z[i],type[i], end=' ', file=fp)
       for j in bonds[i]: print(j, end=' ', file=fp)
       print(file=fp)
+<<<<<<< HEAD
```

Comment:
Another unresolved merge conflict and more of the same below.

Reply:
Resolved this conflict.